### DISCUSSION TITLE
docker not pulling version 4.0.0

### DISCUSSION THREAD

**Discussion by Jitendra1965** [django-recaptcha/django-recaptcha#337] [Q&A] [answered]
I tried to migrate from version 3.0.0 which works fine for my app but when I tried to change it to the latest version 4.0.0 in my django app running in a docker container,  I get the error message " Module django_recaptcha not found" even though it was in the requirements.txt file.  I changed back to the previous version, which works fine.

**Comment by thibaudcolas** [comment]
@Jitendra1965 this is tricky to help you with unless you gave more details – what does something like `pip list` give you after the upgrade? Did you confirm the new version was indeed installed?

**Reply by Jitendra1965** [reply]
Thank you for you response. Executing pip list  in the container did show `django-recaptcha   4.0.0.`  However, when I run the application, it throws an error " `module django-recaptcha NOT found`.  

In the Django settings, I do have:

```
INSTALLED_APPS = [
    'django-recaptcha',
```


```
RECAPTCHA_PUBLIC_KEY = 'mykeys......'
RECAPTCHA_PRIVATE_KEY = 'mykeys.....'
```

ACCOUNT_FORMS = {'login': 'Users.forms.MyCustomLoginForm', 'signup': 'Users.forms.MyCustomSignupForm'}

In the `forms.py`, I have the following import statements:

```
from django_recaptcha.fields import ReCaptchaField
from django_recaptcha.widgets import ReCaptchaV2Checkbox
```


Let me know if you need additional information.

**Reply by rgs258** [reply, marked as answer]
I think it might be a typo in your INSTALLED_APPS, `django-recaptcha` instead of `django_recaptcha`

**Reply by Jitendra1965** [reply]
I apologize.. `rgs258` you were right. The error was fixed by changing it to `django_recaptcha`.
